code/modules/capture_the_flag/ctf_equipment.dm

```diff
@@ -207,7 +197,7 @@
 	///Icon state to be fed into the shielded component
 	var/team_shield_icon = "shield-old"
 	var/max_charges = 150
-	var/recharge_start_delay = 20 SECONDS
+	var/recharge_start_delay = 8 SECONDS
```

Comment:
😕 Not spending longer waiting for your shield to regen than you would dying and respawning makes sense, but 8 seconds might be a bit short. 12 maybe?

Reply:
10 maybe as a compromice? 12 isn't round number and just feels too close too respawning time

Reply:
Remember with respawning time you also need to factor in walking to the front lines in addition to the delay for respawning.